rotkehlchen/chain/ethereum/manager.py

```diff
@@ -658,7 +658,11 @@ def ens_reverse_lookup(self, reversed_addresses: List[ChecksumEthAddress]) -> Di
 
         # Processing human names query output
         for addr, params, human_name_output in zip(filtered_reversed_addresses, human_names_params, human_names_output):  # noqa: E501
-            human_names[addr] = _decode_ens_contract(params=params, result_encoded=human_name_output)  # noqa: E501
+            decoded_name = _decode_ens_contract(params=params, result_encoded=human_name_output)
```

Comment:
This won't work only here. You also call `_decode_ens_contract` above in line 635 and the same treatment should happen there.

But instead what would make sense is to make `_decode_ens_contract` check if it's the zero address and only then return `None`. So change return type of the function too.

Reply:
Yeah okay